packages/components/src/components/text/text.jsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import React from 'react';
+
+const Text = ({ children, size, color, align, weight, lineHeight, className }) => (
```

Comment:
For semantic purposes, better to call this component `Paragraph` instead as it's using a `<p>` tag?

Because when it comes to text, `<p></p>` tags are used to indicate paragraphed text. 

In other cases, when styles are needed to be applied to arbitrary or inline text elements, we use `<span></span>` instead.

Reply:
Can I distinguish them with a prop? because the styling is the same. If it is a paragraph, I use `<p>` otherwise I use `<span>`. What's your idea?

Reply:
Sure, as suggested in the latter part of my comment, we can use the same component for handling both use cases.

By default, a text component should always be inline or `<span>` and not a paragraph `<p>` unless it needs to be. 

The reason for this is because in HTML, `<p>` has a preset of default values in CSS which we don't necessarily want for a simple text usage, if I remember correctly, default `margin-top` and `margin-bottom` for `<p>` is always `1em` and we don't need this for text which don't need to be paragraphed.

We can instead use `is_paragraph` `prop` for ensuring it renders `<p>` instead of `<span>` for when we need the text to be in a paragraph.

Reply:
`<Text as='span' />` or `<Text is_inline />`?  takes to long time to type and read `<Paragraph>`

Reply:
Oskar, you mean better to separate the components com[pletly?

Reply:
Yeah, that's the gist of my second suggested approach, either way the default style for text should be inline.